test/cli.test.mjs

```diff
@@ -1,61 +1,77 @@
 import chai from "chai";
 import shell from "shelljs";
+import { join } from "path";
 
 const expect = chai.expect;
 
 describe("Project builder", function () {
+  // Store absolute paths to the current working directory & path to tmp dir
```

Comment:
these changes aren't required, but when doing local-only GH `test.yml` checks with `act`, i ran into issues with its usage of docker + relative paths. i just make the paths absolute.